src/transformers/modeling_utils.py

```diff
@@ -1927,7 +1927,7 @@ def cuda(self, *args, **kwargs):
     @wraps(torch.nn.Module.to)
     def to(self, *args, **kwargs):
         # Checks if the model has been loaded in 8-bit
-        if getattr(self, "is_quantized", False):
+        if getattr(self, "quantization_method", None) == QuantizationMethod.BITS_AND_BYTES:
             raise ValueError(
                 "`.to` is not supported for `4-bit` or `8-bit` models. Please use the model as it is, since the"
```

Comment:
```suggestion
                "`.to` is not supported for `4-bit` or `8-bit` bitsandbytes models. Please use the model as it is, since the"
```
A tiny suggestion as this is not supported only for bnb models 🙏